assets/base.styl

```diff
@@ -201,6 +202,7 @@ colorItemsPerRow = 7
       padding: 4px 0px
     .ql-picker-item
       height: controlHeight
+	  width: controlWidth
```

Comment:
Can you actually just use controlHeight as the width and keep the whitespace consistent? Thanks!